scripts/release.js

```diff
@@ -178,6 +178,12 @@ if (process.platform == "linux") {
       "com.apple.security.cs.allow-jit": true,
       "com.apple.security.cs.allow-unsigned-executable-memory": true,
       "com.apple.security.cs.disable-library-validation": true,
+
+// Allow dyld environment variables. Needed because Firefox uses
```

Comment:
Firefox?

Reply:
Oops, copied and pasted from here: https://hg.mozilla.org/releases/mozilla-release/rev/4e7be2db653e644365fcf1c569552f1feaf3ee33